ports/nrf/background.c

```diff
@@ -44,14 +46,23 @@
 
 void port_start_background_task(void) {
 }
+
 void port_finish_background_task(void) {
 }
 
-void port_background_task(void) {
+void port_background_tick(void) {
```

Comment:
Is this going to run a lot more often because it's on tick rather than task? Is that OK?